engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/product-repositories.controller.js

```diff
@@ -49,6 +49,7 @@ angular.module('Bastion.products').controller('ProductRepositoriesController',
         $scope.successMessages = [];
         $scope.errorMessages = [];
 
+        $scope.downloadPolicies = [{id: 'immediate', name: 'Immediate'}, {id: 'on_demand', name: 'On Demand'}, {id: 'background', name: 'Background'}];
```

Comment:
Does the name display here need to be translated like you do https://github.com/Katello/katello/pull/5758/files#diff-d985390622f59b70e39a9776ef0b59b9R157 ?  I would think the two could be combined to only store the information once in a simple service.